File: Javascript/8-Kyu/Area-or-perimeter.js
// Area or Perimeter
const areaOrPerimeter = function(l , w) {
  
    return l === w ? l ** 2 : (l + w) * 2;
    
};

// Tests
console.log(areaOrPerimeter(6, 10)); // --> 32
console.log(areaOrPerimeter(3, 3)); // --> 9

File: Python/8-Kyu/Area-or-perimeter.py
# Area or perimeter
def area_or_perimeter(l , w):
    
    return l * w if l == w  else (w + l) * 2

# Test
print(area_or_perimeter(4, 4))  # --> 16
print(area_or_perimeter(6, 10))  # --> 32
